nubis/terraform/dns/outputs.tf

```diff
@@ -0,0 +1,11 @@
+output "zone_id" {
```

Comment:
one little ask. Can you preface all of these outputs with something like 'application_'? This way when we do the work at the account level we will have differentiation that we will not need to adjust downstream.